src/WinRT.Runtime/ComWrappersSupport.net5.cs

```diff
@@ -30,7 +44,7 @@ private static ComWrappers ComWrappers
                     {
                         if (_comWrappers is null)
                         {
-                            _comWrappers = new DefaultComWrappers();
+                            _comWrappers = Instance;
```

Comment:
In both cases, I think you want to do the register first and then set the _comWrappers variable.  As I see for a potential race where if the get is called twice in somewhat parallel to each other, we can end up with one getting the comwrappers instance before it has been registered.